ironfish/src/memPool/feeEstimator.ts

```diff
@@ -41,16 +40,14 @@ export class FeeEstimator {
     this.maxQueueLength = this.numOfRecentBlocks * this.numOfTxSamples
 
     this.queue = []
-    this.chain = options.wallet.chain
```

Comment:
you don't wanna keep this here in the constructor?

Reply:
I removed `chain` as a property of the fee estimator because it's only needed during `init`. Open to restoring it though!